### DISCUSSION TITLE
Cant change the brightness

### DISCUSSION THREAD

**Discussion by Parfuki** [eduardofilo/RG350_adam_image#249] [General]
### What happened?

I cant seem to change the brightness on my anbernic 280V with any of the commands / button combinations.                                  I currently have the newest version of adam image 1.4.2 installed.

Thanks for your help 
Parfuki 

### Device

RG280V

**Comment by neilswann80** [comment]
Just to check... are you definitely holding `POWER ON` and then pressing `LEFT` or `RIGHT`? 
(I have a 280V on 1.4.2 and this works perfectly for me)

Only checking as the old command used to be `POWER ON` and `VOL+` or `VOL-`

**Comment by Parfuki** [comment]
Jeah i already tried it with the power on and the left / right  button combination but its not working for me so im kind of lost on what to do 
thanks for the reply bdw :3

**Comment by neilswann80** [comment]
If you drop out of SimpleMenu to OpenDinguxBeta, within the settings menu launch `GMenu2X` (has a spanner/wrench icon) there's a `Brightness Level` value there you can change by pressing LEFT/RIGHT.  Pressing `START` should exit the app and apply the new brightness level.

Can you check if that route at least is able to change the brightness?

**Comment by Parfuki** [comment]
The question is how do i drop out of the simple menu ?

**Comment by neilswann80** [comment]
- Press `START` to bring up the SimpleMenu settings page
- Change `Default Launcher` to `NO`
- Change `Session` from `Shutdown` to `Quit` and press `A`

That should drop you out of SimpleMenu back into OG OpenDingux.  Then follow my instructions from the post above.

(To return to SinpleMenu, just launch it from the applications section, then undo the changes from this post)

**Comment by Parfuki** [comment]
Jeah it worked that way thank you ;D
